lib/bugsnag/integrations/delayed_job.rb

```diff
@@ -30,7 +30,8 @@ def error(job, error)
               overrides[:job][:queue] = queue
             end
             if job.respond_to?(:attempts)
-              overrides[:job][:attempts] = "#{job.attempts + 1} / #{Delayed::Worker.max_attempts}"
+              max_attempts = job.max_attempts || Delayed::Worker.max_attempts
```

Comment:
Shouldn't this `respond_to?` check whether the job responds to `max_attempts` as well as `attempts`?